tests/tests_fabric/strategies/test_xla.py

```diff
@@ -148,3 +151,32 @@ def tpu_all_gather_fn(strategy):
 def test_tpu_all_gather():
     """Test the all_gather operation on TPU."""
     xla_launch(tpu_all_gather_fn)
+
+
+def tpu_broadcast_master_params_fn(broadcast_master_params, strategy):
+    seed_everything(random.seed())
```

Comment:
This takes an integer

```suggestion
    seed_everything(123)
```

Reply:
Whoops you are right, this should not have `random.seed()`. However, `123` as an input will break the current test since the test currently tests when the broadcast_master_params is set to `True` and `False`. Leaving the input as blank will set the seeds to be different for each device. I will change this when I get back.